ohapi/api.py

```diff
@@ -143,6 +148,11 @@ def upload_file(target_filepath, metadata, access_token, project_member_id,
                       files={'data_file': open(target_filepath, 'rb')},
                       data={'project_member_id': project_member_id,
                             'metadata': json.dumps(metadata)})
+    if not r.status_code == 200:
```

Comment:
Was this tested? From the API code, I thought this would return a 201, not 200.

https://github.com/OpenHumans/open-humans/blob/master/private_sharing/api_views.py#L255